contracts/PatriciaTree/PatriciaTreeProofs.sol

```diff
@@ -5,14 +5,18 @@ pragma experimental "ABIEncoderV2";
 import {Data} from "./Data.sol";
 import {Bits} from "./Bits.sol";
 
+
 /// @title Functions related to checking Patricia Tree proofs
 /// @notice More info at: https://github.com/chriseth/patricia-trie
 contract PatriciaTreeProofs {
   using Bits for uint;
   using Data for Data.Edge;
   using Data for Data.Label;
 
-  function getImpliedRoot(bytes key, bytes value, uint branchMask, bytes32[] siblings) public view returns (bytes32) { // solium-disable-line security/no-assign-params
+  function getImpliedRoot(bytes key, bytes value, uint branchMask, bytes32[] siblings) public // solium-disable-line security/no-assign-params
+  view
+  returns (bytes32)
```

Comment:
I understand we reach the line limit above but can we at least keep `view returns (bytes32)` on the same line?